.travis.yml

```diff
@@ -68,6 +58,16 @@ before_install:
         fi
         sudo systemctl start elasticsearch
     fi
+  - mysql -e 'create database olympia;'
+  - export GOPATH=$HOME/go
+  - export PATH=$HOME/usr/local/go/bin:$GOPATH/bin:$PATH
+  # Unfortunately we require the most recent libmagic-dev version to make use of more recent
+  # JSON detection features which we require for the code-manager related APIs
+  - echo "deb http://archive.ubuntu.com/ubuntu eoan main" | sudo tee -a /etc/apt/sources.list
+  - sudo apt-get update -qq
+  - sudo apt-get install -t eoan libmagic-dev libmagic1 libmagic-mgc
```

Comment:
I am wondering if we still need all of that 🤔

Reply:
My bad, you removed it in 09bbe0a61661dfb53927c3ab42af0fddc38614d8 but I accidentally added it back when merging ES 6 support in ec0d8f9293d8600afffcf9136a2c0875197846cd - so when I did this PR I moved it without realizing it should have been removed already...

Reply:
https://github.com/mozilla/addons-server/pull/15157